src/otp/otp.controller.spec.ts

```diff
@@ -0,0 +1,58 @@
+import { ConfigModule } from "@nestjs/config";
+import { Test, TestingModule } from "@nestjs/testing";
+import { expect } from "chai";
+import { repositoryMockFactory } from "../user/user.service.spec";
+import { KycService } from "../services/KycService";
+import { UserService } from "../user/user.service";
+import { OtpController } from "./otp.controller";
+import { EmailService } from "../services/EmailService";
```

Comment:
Same here for `services`, is there not an `index.ts` file that lets you combine this with line 5?

Reply:
good point, but not gonna do it on this ticket, its a bug refactor already